src/utils/is-image-url.js

```diff
@@ -0,0 +1,15 @@
+import { parse } from 'url';
```

Comment:
Is this similar to [`URL()`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL)?